routes/classes.js

```diff
@@ -0,0 +1,63 @@
+const express = require("express")
+const cors = require("cors")
+const School = require("../School")
+const Student = require("../Student")
+let classes = express.Router()
+classes.use(cors())
+let mySchool = new School();
+classes.get("/", (request, response) => {
+    response.json(mySchool)
+})
+
+classes.post("/", (request, response) => {
+    let classes = request.body.classes;
+    let teacher = request.body.teacher;
+    if(classes.length===0|| teacher.length===0){
+        response.json({message: "Please fill out all the information or Class already exists"})
```

Comment:
You are not actually checking if class already exists. You are just checking if all the information has been filled out. 
1. As an option you could check if all the required fields of the form have been filled out on the frontend and not even make the axios call unless everything is done correctly by the user.
2. You can use a middleware function to check if class exists, if it does - send a response with a specific error message.  If it doesn't you would move on to the `next()` function.